utils/mstpctl-utils-functions.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
```

Comment:
Should probably have a blank line after the shebang and before mstpctl_parse_ports() ?